src/types/models/tokens.ts

```diff
@@ -44,6 +50,7 @@ interface Token<DataType = Primitive> extends TokenBase {
   encryption?: TokenEncryption;
   searchIndexes?: string[];
   fingerprintExpression?: string;
+  mask?: TokenMask;
```

Comment:
seems like we can make use of the existing `TokenData<DataType>` type here. maybe we can just rename it

Reply:
i didn't reuse it because it would allow types other than strings, and mask can be a complex object but only of strings

Reply:
mask works with numbers and bools as well. in either case, you can still make use of that type with `TokenData<string>`